iptb/iptb.go

```diff
@@ -11,6 +11,7 @@ import (
 	docker "github.com/ipfs/iptb-plugins/docker"
 	local "github.com/ipfs/iptb-plugins/local"
 	localp2pd "github.com/ipfs/iptb-plugins/localp2pd"
+	localbtfs "github.com/ipfs/iptb-plugins/localbtfs"
```

Comment:
`github.com/TRON-US/iptb-plugins-1/localbtfs`? the iptb-plugins original repo does not have localbtfs